src/compiler/utilities.ts

```diff
@@ -4284,25 +4284,22 @@ export function forEachImportClauseDeclaration<T>(node: ImportClause, action: (d
 }
 
 /** @internal */
-export function hasQuestionToken(node: Node) {
-    if (node) {

```

Comment:
Is this behavior change intentional? It's not annotated to take `Node`, but I would verify we didn't put this here to handle some silly crash. (I would think we should do this change though)

Reply:
yes, I intentionally made this change. For all current references to `hasQuestionToken`, a `Node` is always passed in and can't be undefined. I wasn't able to trace the history of changes for this function, but I'm guessing that other things in the function were deleted, and this check was accidentally left in.